wireapp/wire-ios-mono
# chore: bump AVS 9.6.12 - WPB-6834

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-6834" title="WPB-6834" target="_blank"><img alt="Task" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10818?size=medium" />WPB-6834</a>  [iOS] Update to AVS 9.6.12 to include test results
  </td></table>
  <br />
 
























# What's new in this PR?

In order to fix a bug that happened on Web, we bump AVS and call setMLSConferenceInfo inside the `req_client` handler when it's an mls conversation.

## How to test

on a MLS conv between A and B :
- A calls B
- B ignore the call and kill the app
- B launches the app and tap on call button before the rejoin button appears

call should work (expecially with audio)
